modules/elt/pages/use-airbyte-to-load-data-from-external-sources-to-teradata-vantage.adoc

```diff
@@ -0,0 +1,200 @@
+
+* The link to the spreadsheet to pull the data from is to be added in `Spreadsheet Link`.
+
+
+image::{dir}/configuring_source_gsheet_airbyte.png[Configuring the source in Airbyte]
+
+[NOTE]
+====
+For more details, please refer link:https://docs.airbyte.com/integrations/sources/google-sheets/#:~:text=For%20Airbyte%20Open%20Source%3A[Setting Google Sheets as Source Connector in Airbyte Open Source] 
+====
+
+* Click Set up source, if configuration is correct, you will get the message `All connection tests passed!`
+
+
+=== Setting the Destination Connection
+* Assuming you want to create a fresh new connection with `Teradata Vantage`, Select `Teradata Vantage` as destination type under Set up the destination section.
+* Add the `Host`, `User`, and `Password`. This is the same as `Host`, `Username`, and `Password` respectively that are used to create the Teradata Vantage instance in the Clearscape Analytics Environment.
+* Default schema name is optional, it provides a default name, however, you can provide a name that makes more sense. Here we have provided `gsheet_airbyte_td`.
+
+image::{dir}/configuring_destination_teradata_airbyte.png[Configuring the destination Teradata in Airbyte]
+
+
+* Click Set up destination, if the configuration is correct, you will get the message `All connection tests passed!`
+
+
+[NOTE]
+====
+You might get a configuration check failed error. Make sure your Teradata Vantage instance is running properly before making a connection through Airbyte. 
+====
+
+=== Configuring Data Sync
+Namespace is a group of streams(tables) in a source or destination. A schema in a relational database system is an example of a namespace. In a source, the namespace is the location from where the data is replicated to the destination. In a destination, the namespace is the location where the replicated data is stored in the destination.
+For more details please refer link:https://docs.airbyte.com/understanding-airbyte/namespaces/[Airbyte Namespace]
+
+
+image::{dir}/namespaces_in_destination.png[Namespaces in the destination]
+
+
+In this use case the destination is a database, so the namespace is the default schema `gsheet_airbyte_td` we defined when we configured the destination, and the stream name is a table that is mirroring the name of the spreadsheet in the source, which is `sample_employee_payrate` in this case. Other type of sources and destinations might have a different layout, in this example, Google sheets as source does not support a namespace, and since it is the single spreadsheet connector, it only supports one stream (the active spreadsheet).
```

Comment:
```suggestion
In our example the destination is a database, so the namespace is the default schema `gsheet_airbyte_td` we defined when we configured the destination. The stream name is a table that is mirroring the name of the spreadsheet in the source, which is `sample_employee_payrate` in this case. + 
Other type of sources and destinations might have a different layout. In this example, Google sheets as source does not support a namespace, as well, since we are using the single spreadsheet connector, it only supports one stream (the active spreadsheet).
```